File: java/src/main/java/com/experian/dvs/client/address/format/components/LocalityItem.java
package com.experian.dvs.client.address.format.components;

import java.util.Objects;

public class LocalityItem {
    private final String name;
    private final String code;
    private final String description;

    // Constructor
    public LocalityItem(String name, String code, String description) {
        this.name = Objects.toString(name, "");
        this.code = Objects.toString(code, "");
        this.description = Objects.toString(description, "");
    }

    // Getters
    public String getName() {
        return name;
    }

    public String getCode() {
        return code;
    }

    public String getDescription() {
        return description;
    }
}

File: csharp/DVSClient/DVSClient/address/format/components/LocalityItem.cs
namespace DVSClient.Address.Format.Components
{
    public class LocalityItem
    {
        public string? Name { get; }
        public string? Code { get; }
        public string? Description { get; }

        public LocalityItem(string? name, string? code, string? description)
        {
            Name = name;
            Code = code;
            Description = description;
        }
    }
}